react-form-controls/my-app/src/RegistrationFormUncontrolled.jsx

```diff
@@ -0,0 +1,28 @@
+export default function RegistrationFormUncontrolled() {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const usrnm = document.querySelector('#usrnm').value;
+    const psswrd = document.querySelector('#psswrd').value;
+    const formData = new FormData();
+    formData.append('username:', usrnm);
+    formData.append('password:', psswrd);
+    for (const pair of formData.entries()) {
```

Comment:
You can simplify here and use `Object.fromEntries(formData.entries()) `  to get the individual field values instead